### DISCUSSION TITLE
What color is the bot playing as?

### DISCUSSION THREAD

**Discussion by dudecon** [SebLague/Chess-Challenge#187] [Q&A] [answered]
Do I need to check IsWhiteToMove every time Think is called to determine my color? Or maybe it doesn't matter? Except that, to use the bitboards, I need to differentiate between my pieces and the opponent's pieces. Shouldn't the bitboards be "mine" and "opponents" instead of "black and white"?
Or, am I thinking about this all sideways?

**Comment by tomc128** [comment, marked as answer]
Every game is different, i.e. you'll be black one game and could be white the next. So far, checking `board.IsWhiteToMove` was the best way I could find to determine the colour; it'll obviously be the same value throughout the game.
